contracts/KyberNetwork.sol

```diff
@@ -40,8 +40,9 @@ contract ReentrancyGuard {
 contract KyberNetwork is Withdrawable, Utils2, KyberNetworkInterface, ReentrancyGuard {
 
     enum ReserveType {NONE, PERMISSIONED, PERMISSIONLESS}
-    bytes internal empty;
-    bytes internal permHint = "PERM";
+    bytes internal constant EMPTY_HINT = "";
+    bytes internal constant PERM_HINT = "PERM";
```

Comment:
maybe this should be public? victor could use it.

Reply:
OK
will do